mvn clean compile fails with org.eclipse.xtext.generator:[2.11.0] resolution error

When executing `mvn clean generate-sources` or `mvn clean compile` I am getting the following error message:

```bash
~/workspace/xcore-maven-example $ mvn clean generate-sources                   
[INFO] Scanning for projects...
[INFO]                                                                         
[INFO] ------------------------------------------------------------------------
[INFO] Building xcore-maven-example 0.2.0-SNAPSHOT
[INFO] ------------------------------------------------------------------------
[INFO] 
[INFO] --- maven-clean-plugin:2.6.1:clean (default-clean) @ xcore-maven-example ---
[INFO] 
[INFO] --- build-helper-maven-plugin:1.9.1:add-source (add-source) @ xcore-maven-example ---
[INFO] Source directory: /home/harry/workspace/xcore-maven-example/src/main/java-gen added.
[INFO] 
[INFO] --- xtext-maven-plugin:2.11.0:generate (default) @ xcore-maven-example ---
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 0.966 s
[INFO] Finished at: 2017-06-28T01:20:16+02:00
[INFO] Final Memory: 8M/153M
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.eclipse.xtext:xtext-maven-plugin:2.11.0:generate (default) on project xcore-maven-example: Execution default of goal org.eclipse.xtext:xtext-maven-plugin:2.11.0:generate failed: Plugin org.eclipse.xtext:xtext-maven-plugin:2.11.0 or one of its dependencies could not be resolved: Failed to collect dependencies at org.eclipse.xtext:xtext-maven-plugin:jar:2.11.0 -> org.eclipse.xtext:org.eclipse.xtext.generator:jar:[2.11.0]: No versions available for org.eclipse.xtext:org.eclipse.xtext.generator:jar:[2.11.0] within specified range -> [Help 1]
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/PluginResolutionException
```

--- a/pom.xml
+++ b/pom.xml
@@ -151,6 +151,11 @@
                         <version>${xtext-version}</version>
                     </dependency>
                     <dependency>
+                        <groupId>org.eclipse.xtext</groupId>
+                        <artifactId>org.eclipse.xtext.generator</artifactId>
+                        <version>${xtext-version}</version>
+                    </dependency>
+                    <dependency>
                         <groupId>org.eclipse.emf</groupId>
                         <artifactId>org.eclipse.emf.codegen.ecore.xtext</artifactId>
                         <version>${ecore-xtext-version}</version>

